### PR TITLE
Docs: Brief overview of the server components.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: docs/administration.rst [hunk 1]
```diff
@@ -112,3 +133,18 @@ once they have been started, users can use ``bodhi composes list`` or they can v
 * **signing_repo**: The composer is waiting on the repository to be signed.
 * **cleaning**: The composer is cleaning up old composes. This state only occurs if
   ``clean_old_composes`` is set to True in the settings.
+
+
```
Comment (tdawson) [inline docs/administration.rst:145]: Greenwave Handler

Bodhi's greenwave handler watches for messages sent from greenwave about bodhi updates that have been tested.  When a message is recieved, the appropriate update is updated with the greenwave test status.

Comment (bowlofeggs) [reply docs/administration.rst:145]: I'm adding this patch:

```
diff --git a/docs/administration.rst b/docs/administration.rst
index 1eb4e5ca..3485b585 100644
--- a/docs/administration.rst
+++ b/docs/administration.rst
@@ -135,6 +135,14 @@ once they have been started, users can use ``bodhi composes list`` or they can v
   ``clean_old_composes`` is set to True in the settings.
 
 
+Greenwave Handler
+^^^^^^^^^^^^^^^^^
+
+Bodhi's greenwave handler watches for messages sent from greenwave about bodhi updates that have
+been tested. When a message is recieved, the appropriate update is updated with the greenwave test
+status.
+
+
 Signed Handler
 ^^^^^^^^^^^^^^
```

**Comment by bowlofeggs** [comment]
I could use review on this patch.

**Comment by cverna** [comment]
We might also want to add the consumer that automatically creates updates. Other than that looks good to me.

**Comment by bowlofeggs** [comment]
I am applying this patch:

```
diff --git a/docs/administration.rst b/docs/administration.rst
index 3485b585..e4884fb6 100644
--- a/docs/administration.rst
+++ b/docs/administration.rst
@@ -103,6 +103,14 @@ Message Consumers
 The following sections document the list of messaging consumers.
 
 
+Automatic updates
+^^^^^^^^^^^^^^^^^
+
+This consumer will create updates automatically when builds are tagged to a Release's candidate tag,
+if the Release associated with that tag is configured to do so by having its
+``create_automatic_updates`` boolean set to True.
+
+
 Composer
 ^^^^^^^^
```

**Review by cverna — APPROVED** [review]

**Pull request merged**